src/index.js

```diff
@@ -97,6 +97,32 @@ module.exports = {
         ignoreRestSiblings: true,
       },
     ],
+    "@typescript-eslint/camelcase": "off",
+    "@typescript-eslint/naming-convention": [
```

Comment:
Can you summarize what it does, it's quite time intentes to get everything together from the docs

Reply:
Will mention that in the docs for sure, but here's a short summary:
- a default name for everything is in `camelCase`
- if it's a member or a variable we also accept `PascalCase` - this is for classes/components
- for the properties, we also allow `snake_case` - this covers a bunch of external properties that are expected to be in `snake_case` as well as 99.9% of our BEM convention that we use with classnames
- for the types, enums and interfaces we want to use `PascalCase` only

Reply:
Thanks, sound good to me.